src/app/shared/models/comics/comics.ts

```diff
@@ -1,4 +1,6 @@
-
+import { Url } from "../common/common"; 
```

Comment:
add all imports bellow the same line 
import { Url, Image, SeriesSummary } from "../common/common";